### ISSUE TITLE
More responsive layout

### ISSUE THREAD

**Issue by barryross** [codetodesign/NYUAMS#663]
Related to: https://github.com/codetodesign/NYUAMS/issues/475

I think https://github.com/codetodesign/NYUAMS/issues/475 is part of a larger issue of the app not being very responsive.  Perhaps we should: 

a) stack the pagination beneath the buttons when the browser is below a certain size to get rid of the overlap?  

b) gracefully shrink the columns so they share the remain side-by-side when the window size is reduced 

<img width="901" alt="screen shot 2017-01-03 at 10 08 48 pm" src="https://cloud.githubusercontent.com/assets/2281667/21633101/12df404e-d201-11e6-9e7f-059e897567a0.png">

**Comment by gosha1128** [comment]
Do you think maybe it's best to rescope the effort behind this issue ?

**Comment by barryross** [comment]
I would be inclined to resolve #475 by making the nav bar stack, rather than consolidating the links to 3 items - I think it’d be a more user-friendly and easier solution to implement.    

Additionally, I think #663 may be a larger scale effort to make the app more responsive in general, and would require some time to go through and re-scope what it’d take to improve the experience on smaller screens.  That being said, I’m not sure if the lack of responsiveness is causing that many issues for the client currently, other than the pagination.  

Does that help clarify?

**Comment by gosha1128** [comment]
Yes.  Do what you can now...and then rescope the effort on 663.  Update Leo
regarding the invoice accordingly...

**Comment by barryross** [comment]
Gotcha…actually #663 hasn’t been on any invoice/estimate yet…so I think it would just be scoping it and not re-scoping?

**Comment by gosha1128** [comment]
Yes.  Leo shall we add to the current batch ?

**Comment by leobardoramon** [comment]
Andrew,

Would you like this to be (a) fixed sooner or (b) added to the next round of bug fixes ( aka DEVN-003 ) for Feb/Mar ?

**Comment by NYU-AMS** [comment]
Make it a B fix
